tests/utils/dut.py

```diff
@@ -38,7 +38,7 @@ def dut_cleanup():
 
     output = TestRun.executor.run('pgrep iotrace')
     if output.stdout != "":
-        TestRun.executor.run(f'kill -2 {output.stdout}')
+        TestRun.executor.run(f'kill -9 {output.stdout}')
```

Comment:
Maybe we can use method kill_tracing from plugin?

Reply:
Kill tracing uses pid setup from start tracing. With this usage we want to scan any iotrace instances not killed, so we don't have a corresponding plugin instance.